doc/source/Resources/PyAEDTInstallerFromDesktop.py

```diff
@@ -10,18 +10,25 @@
 
 
 def run_pyinstaller_from_c_python(oDesktop):
-    # This is called when run from IronPython
+    # Iron Python script to create the virtual environment and install PyAEDT
+    # Get AEDT information
     version = oDesktop.GetVersion()[2:6].replace(".", "")
+    # From AEDT 2023.2 the installed CPython version is 3.10
     python_version = "3.10"
     if version <= "231":
         python_version = "3.7"
+    # AEDT installation root
     edt_root = os.path.normpath(oDesktop.GetExeDir())
+    # CPython interpreter executable
     if is_windows:
         python_exe = os.path.normpath(os.path.join(edt_root, "commonfiles", "CPython", python_version.replace(".", "_"),
                                                    "winx64", "Release", "python", "python.exe"))
     else:
         python_exe = os.path.normpath(os.path.join(edt_root, "commonfiles", "CPython", python_version.replace(".", "_"),
                                                    "linx64", "Release", "python", "runpython"))
+
+    # Launch this script again from the CPython interpreter, this will call install_pyaedt method
+    # This method is creating a virtual environment and installing pyaedt and its dependencies
```

Comment:
```suggestion
    # Launch this script again from the CPython interpreter. This calls the ``install_pyaedt()`` method,
    # which creates a virtual environment and installs PyAEDT and its dependencies
```
Even comments should follow our style guide.